### ISSUE TITLE
Struct validation: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by tsdevelopment** [gookit/validate#276]
**System (please complete the following information):**

 - OS: `linux/amd64`
 - GO Version: `go1.23.3 linux/amd64`
 - Pkg Version: `v1.5.2` 

**Describe the bug**

When running struct validation the following error occurs:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xd0 pc=0x83dfa5]

goroutine 1 [running]:
github.com/gookit/validate.(*StructData).parseRulesFromTag.func1({0x0?, 0x0?, 0xc000025a70?}, {0x0, 0x0}, {0x0, 0x0}, 0x0)
	/go/pkg/mod/github.com/gookit/validate@v1.5.2/data_source.go:300 +0xc5
github.com/gookit/validate.(*StructData).parseRulesFromTag(0xc0002140e0, 0xc0000b8420)
	/go/pkg/mod/github.com/gookit/validate@v1.5.2/data_source.go:435 +0x1c6
github.com/gookit/validate.(*StructData).Create(0xc0002140e0, {0xc000025a40, 0x1, 0xc0001d5590?})
	/go/pkg/mod/github.com/gookit/validate@v1.5.2/data_source.go:255 +0x8f
github.com/gookit/validate.mustNewValidation({0x9dfe10, 0xc0002140e0}, {0x0, 0x0})
	/go/pkg/mod/github.com/gookit/validate@v1.5.2/validate.go:283 +0xbd
github.com/gookit/validate.Struct({0x87b180?, 0xc0001f7e68?}, {0x0, 0x0, 0xd4f7a0?})
	/go/pkg/mod/github.com/gookit/validate@v1.5.2/validate.go:267 +0x3c
main.main()
	main.go:17 +0x5f
exit status 2

```

**To Reproduce**

```go
package main

import (
	"fmt"

	"github.com/gookit/validate"
)

func main() {
	type user struct {
		Name string `validate:"required"`
		Age  int    `validate:"required_if:Name,lee"`
	}

	u := &user{Name: "lee"}
	v := validate.Struct(u)

	if v.Validate() { // validate ok
		// do something ...
	} else {
		fmt.Println(v.Errors)               // all error messages
		fmt.Println(v.Errors.One())         // returns a random error message text
		fmt.Println(v.Errors.OneError())    // returns a random error
		fmt.Println(v.Errors.Field("Name")) // returns error messages of the field
	}
}

```

**Comment by inhere** [comment]
shoud fixed in https://github.com/gookit/validate/pull/260

**Comment by tsdevelopment** [comment]
Yes, seems to be fixed in #260. Didn's see that, sorry! But there was no release since January. Are there any plans for a new release?

**Comment by inhere** [comment]
> Yes, seems to be fixed in #260. Didn's see that, sorry! But there was no release since January. Are there any plans for a new release?

Yes, A version will be released soon
